assets/css/style.less

```diff
@@ -37,8 +37,13 @@ textarea {
         width: 50px;
         padding: 0.1em;
         .logotype {
-        }
     }
+  }
+  .navbar-user-gravatar {
```

Comment:
@oBlissing Please do the PyCharm + ⌥⌘L and commit again!

Also, I don't think the previous line-40 brace was a "stray" one.  The entire `.logotype` block is!

Reply:
Good catch 🥇